addons/core/tests/integration/helpers/time-remaining-test.js

```diff
@@ -13,19 +13,26 @@ module('Integration | Helper | time-remaining', function (hooks) {
   setupRenderingTest(hooks);
   setupIntl(hooks, 'en-us');
 
-  test('it calculates the correct time remaining', async function (assert) {
-    assert.expect(1);
-
+  test('it calculates the correct time remainin with less than 24 hours', async function (assert) {
     this.set('expirationTime', new Date(Date.now() + 1000 * 60 * 60 * 24));
 
     await render(hbs`{{time-remaining this.expirationTime}}`);
 
     assert.strictEqual(this.element.textContent.trim(), '23:59:59 remaining');
   });
 
-  test('it handles negative time remaining', async function (assert) {
-    assert.expect(1);
+  test('it calculates the correct time remaining with more than 24 hours', async function (assert) {
```

Comment:
Could we also have a test for more than a week to see how it's formatted?

Reply:
Done.